arch/arm/core/cortex_m/prep_c.c

```diff
@@ -24,6 +24,45 @@
 #include <arch/arm/cortex_m/cmsis.h>
 #include <string.h>
 
+extern uint32_t _interrupt_stack;
```

Comment:
Also, I wonder whether we could include stack.h here (so we get the declaration of _interrupt_stack)

Reply:
I will check it.